lib/npg_pipeline/product/release/irods.pm

```diff
@@ -15,6 +16,8 @@ our $VERSION = '0';
 Readonly::Scalar my $THOUSAND                    => 1000;
 Readonly::Scalar my $PRODUCTION_IRODS_ROOT       => q[/seq];
 Readonly::Scalar my $IRODS_REL_ROOT_NOVASEQ_RUNS => q[illumina/runs];
+Readonly::Scalar my $IRODS_PP_CONF_KEY           =>
```

Comment:
Not sure how I feel about aliasing a constant. Does the composed `release` role not import this constant into the namespace?